docs/user/UserGuide.md

```diff
@@ -212,32 +211,10 @@ Is satisfied if `field`'s value is equal to `value`
 
 Is satisfied if `field`'s value is in the set `values`
 
-<div id="predicate-null"></div>
-
-### `null` _(field)_
-
-```javascript
-{ "field": "price", "is": "null" }
-```
-
-Is satisfied if `field` is null or absent.
-
-<div id="predicate-oftype"></div>
-
-### `ofType` _(field, value)_
-
-```javascript
-{ "field": "price", "is": "ofType", "value": "string" }
-```
-
-Is satisfied if `field` is of type represented by `value` (valid options: `decimal`, `integer`, `string`, `datetime`, `ISIN`, `SEDOL`, `CUSIP`, `RIC`, `firstname`, `lastname` or `fullname`)
-
-<div id="predicate-setfromfile"></div>
-
-### `setFromFile` _(field, value)_
+Alternatively, sets can be populated from files.
 
 ```javascript
-{ "field": "country", "is": "setFromFile", "value": "countries.csv" }
+{ "field": "country", "is": "inSet", "file": "countries.csv" }
 ```
 
 Populates a set from the new-line delimited file (with suffix `.csv`), where each line represents a string value to load.
```

Comment:
> The file should be location in the same directory as the jar, or in the directory explicitly specified using the command line argument `--set-from-file-directory`, and the name should match the `value` with `.csv` appended.

Not exactly where you were working but the directory must be specified as C:\a\b\c\ . The form C:\a\b\c (with no final slash) will fail. Is this intended behaviour?

Reply:
Can you raise a bug for this

Reply:
done #1196